test/test_fields.py

```diff
@@ -47,3 +47,9 @@ def test_render_part(self):
         field = RequestField('somename', 'data')
         param = field._render_part('filename', u('n\u00e4me'))
         self.assertEqual(param, "filename*=utf-8''n%C3%A4me")
+
+    def test_render_unicode_bytes_py2(self):
+        if not PY3:
```

Comment:
We have [test decorators](https://github.com/shazow/urllib3/blob/master/test/__init__.py#L37-L79) for this.